wordle/src/main/kotlin/domain/Game.kt

```diff
@@ -0,0 +1,27 @@
+package domain
+
+import java.time.LocalDateTime
+
+class Game(
+    private val wordList: WordList // TODO: 이름 개선
+) {
+    private val submittedWords = mutableMapOf<Word, LetterCompareResults>()
+    val letterCompareResultsList // TODO: 이름 개선
+        get() = submittedWords.values
+
+    // TODO: 리팩토링
+    fun submitWord(text: String, now: LocalDateTime): LetterCompareResults {
+        val submittedWord = Word(text)
+        if (!wordList.contains(submittedWord)) {
+            throw IllegalArgumentException("단어 리스트에 존재하는 단어를 제출해야합니다.")
+        }
+
+        val todayWordPicker = TodayWordPicker(wordList)
+        val todayWord = todayWordPicker.pick(now)
+
+        val letterCompareResult = submittedWord.compareWithCorrectAnswer(todayWord)
```

Comment:
```suggestion
        val letterCompareResults = submittedWord.compareWithCorrectAnswer(todayWord)
```

메서드 반환값이 `LetterCompareResults`인데 왜 `LetterCompareResult`를 반환하고 있지?? 하고 착각해서 살짝...

Reply:
ㅋㅋㅋㅋ ㅠ 코드가 난리났네요